src/measure_theory/integral/interval_integral.lean

```diff
@@ -213,20 +213,20 @@ integrable_on f (Ioc a b) μ ∧ integrable_on f (Ioc b a) μ
   only if it is integrable on `Ioc (min a b) (max a b)` with respect to `μ`. This is an equivalent
```

Comment:
should we update the docstrings for these or leave them as `Ioc .. ..`?

Reply:
I'll do it.